src-electron/generator/matter/app/zap-templates/common/attributes/Accessors.js

```diff
@@ -76,6 +79,8 @@ async function accessorGetterType(attr) {
     type = `DataModel::Nullable<${type}> &`;
   } else if (mayNeedPointer) {
     type = `${type} *`;
+  } else if (mayNeedReference) {
```

Comment:
Would the following give the same result?
else {
type = `${type} &`;
}

Reply:
@brdandu For now, yes, unless we add some types that need neither pointer not reference.  I can do that.